### PR TITLE
Migrate to state views

### PR DESCRIPTION
**Pull request by ppepos**
Please test-run this as it was difficult to try and test every little feature as all of the pathing and controller resolution was changed.

Thanks <3

Signed-off-by: Philippe Pepos Petitclerc ppeposp@gmail.com

### CODE REVIEW

**Comment by Morriar** [comment]
Nothing is displaying on missions-dev.nitlanguage.org.

Reason: 

```
Failed to instantiate module ngRoute due to:
Error: [$injector:nomod] http://errors.angularjs.org/1.5.5/$injector/nomod?p0=ngRoute
```

Maybe try a `git grep ngRoute` to remove them all.

File: www/directives/player/player.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class='container-fluid'>
-	<panel-404 ng-if='playerCtrl.error' />
+	<panel-404 ng-if='vm.error' />
```
Comment (Morriar) [inline www/directives/player/player.html:2]: I'm not against the `vm` controller name but in my experience, having all your controllers named `vm` makes a huge deal with nested controllers.

File: www/javascripts/players.js [hunk 2]
```diff
@@ -18,8 +18,8 @@
 	angular
 		.module('players', ['ngSanitize', 'model'])
 
-		.controller('PlayerHome', ['$routeParams', '$scope', function($routeParams, $scope) {
-			$scope.playerId = $routeParams.login;
+		.controller('PlayerHome', ['$stateParams', '$scope', function($stateParams, $scope) {
+			$scope.playerId = $stateParams.login;
 		}])
 
 		.controller('PlayerAuth', ['$routeParams', '$rootScope', '$scope', function($routeParams, $rootScope, $scope) {
```
Comment (Morriar) [inline www/javascripts/players.js:25]: I see a `routeParams` here

**Review by Morriar — CHANGES_REQUESTED** [review]

**Comment by privat** [comment]
I close the PR so that http://missions-dev.nitlanguage.org can be functionnal again, reopen it when the `routeParams` are OK.

**Comment by ppepos** [comment]
Wow can't reopen because `-f`..................

Opening a new one. =/

**Pull request closed without merging**